.github/workflows/test-and-deploy-alpha.yml

```diff
@@ -3,27 +3,96 @@ name: test-and-deploy-alpha
 on: [push]
 
 jobs:
-  install-deps:
+  prepare:
     runs-on: [ubuntu-latest]
     steps:
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache node_modules and artifacts
+        uses: ./.github/actions/restore-cache
+
+      - name: Set up Node
+        uses: ./.github/actions/setup-node
+
       - name: Install dependencies
-        run: npm i
-      - name: List all dependencies
-        shell: bash
-        run: npm list
+        run: npm ci --legacy-peer-deps
 
-  unit-tests:
-    needs: install-deps
+      - name: Build essentials
+        uses: ./.github/actions/build-essential
+
+  prepare-win32:
+    runs-on: [windows-latest]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache node_modules and artifacts
+        uses: ./.github/actions/restore-cache
+
+      - name: Set up Node
+        uses: ./.github/actions/setup-node
+
+      - name: Install dependencies
+        run: npm ci --legacy-peer-deps
```

Comment:
does it need the `--legacy-peer-deps` flag?

Reply:
It does as some dependencies of `slice-machine-ui` don't (officially?) support React 19, making the install fail otherwise.

Reply:
`@reach/menu-button` was giving me trouble without `--legacy-peer-deps`. It only officially supports React 17, so you'll get a peer deps error from npm without the flag.